pkg/errcode/errcode.go

```diff
@@ -349,6 +352,12 @@ const (
 
 	// ErrAddingDeletedMutatingWebhook indicates an error occurred when OSM Reconciler failed to add a deleted mutating webhook
 	ErrAddingDeletedMutatingWebhook
+
+	// ErrUpdatingValidatingWebhook indicates an error occurred when OSM Reconciler failed to update the validating webhook
+	ErrUpdatingValidatingWebhook
+
+	// ErrAddingDeletedValidatingWebhook indicates an error occurred when OSM Reconciler failed to add a deleted validating webhook
+	ErrAddingDeletedValidatingWebhook
```

Comment:
The naming and comment can be improved as follows:
```suggestion
	// ErrReconcilingDeletedValidatingWebhook indicates an error occurred when OSM Reconciler failed to reconcile a deleted validating webhook config
	ErrReconcilingDeletedValidatingWebhook
```

Same applies to the mutating webhook constant and comments.